### ISSUE TITLE
ComboBox.ItemTemplate is ignored in the flyout when using MaterialComboBoxStyle (v1) 

### ISSUE THREAD

**Issue by jeanplevesque** [unoplatform/Uno.Themes#900]
## Current behavior

Using `ComboBox.ItemTemplate` only has an effect on the selected item.
It's ignored in the list popup.

```xaml
<ComboBox.ItemTemplate>
  <DataTemplate>
    <TextBlock Text="{Binding Converter={StaticResource EnumToString}}"
	   Foreground="Red" />
  </DataTemplate>
</ComboBox.ItemTemplate>
```

Notice how the red color **is not** applied everywhere.

![material-combobox-bug](https://user-images.githubusercontent.com/39710855/208759836-aedf9bd5-1399-4ff1-a477-da98103441a2.gif)

## Expected behavior

Notice how the red color **is** applied everywhere.

![combobox-correct](https://user-images.githubusercontent.com/39710855/208759998-60bb672d-223e-4482-9ef3-21a25793e836.gif)

## How to reproduce it (as minimally and precisely as possible)

Use a ComboBox that has the style `MaterialComboBoxStyle` from MaterialV1 and set a custom `ItemTemplate`.

```xml
<ComboBox Style="{StaticResource MaterialComboBoxStyle}">
  <ComboBox.ItemTemplate>
    <DataTemplate>
      <TextBlock Text="Bob"
		   Foreground="Red" />
    </DataTemplate>
  </ComboBox.ItemTemplate>
</ComboBox>
```

## Environment

Nuget Package: Uno.Material

Package Version(s): 1.3.3

Affected platform(s):

- [x] iOS
- [x] Android
- [x] WebAssembly
- [x] UWP
- [x] MacOS

**Comment by Xiaoy312** [comment]
fixed in #849

**Comment by Xiaoy312** [comment]
fixed in https://github.com/unoplatform/Uno.Themes/pull/849
